src/components/App.jsx

```diff
@@ -2,22 +2,22 @@ import React, { Component } from "react";
 import { Switch, BrowserRouter as Router, Route } from "react-router-dom";
 import importedComponent from "react-imported-component";
 
-import Home from "./Home";
-import Loading from "./Loading";
+import Home from "./Home.jsx";
```

Comment:
it is strange that u specify suffix, add it at least to TODO the list after vite migration

Reply:
It was created by IDEA, not by me, but it works. I'll add it to the list.